services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/user_services_preferences/_errors.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+
+
+class BaseServicesPreferencesError(Exception):
+    ...
+
+
+class DestinationIsNotADirectoryError(BaseServicesPreferencesError):
+    def __init__(self, destination_to: Path) -> None:
```

Comment:
MINOR: highly recommend to use `PydanticErrorMixin` as base. We are all using that approach since a while.
As a reminder, please search for `PydanticErrorMixin` in the codebase